index.js

```diff
@@ -155,6 +155,41 @@ async function fetchChunks(prefix, levelsRemaining, stream) {
     return;
   }
 
+  // Async fetch the last level
+  if (process.env.QUICK_MODE && levelsRemaining == 1) {
+    let promises = [];
+    for (let i = 0; i < 256; i++) {
+      promises.push(fetchChunks(prefix + i.toString(16).padStart(2, "0"), levelsRemaining - 1, stream));
+    }
+    await Promise.all(promises);
+  } else {
+    for (let i = 0; i < 256; i++) {
+      await fetchChunks(prefix + i.toString(16).padStart(2, "0"), levelsRemaining - 1, stream);
+    }
+  }
+}
+
+async function fetchChunksAdvance(prefix, levelsRemaining, stream, exclude_prefix) {
```

Comment:
Please help test and verify this.

And better to support arrays by changing `exclude_prefix` to `exclude_prefixes` here.